app/js/components/identifiers/addIdentifier.jsx

```diff
@@ -0,0 +1,231 @@
+                                        onChange={this.handleCreateChange}
+                                    />
+                                    {(addError.includes('identifier,') ||
+                                        !(addError.includes('identifierType') ||
+                                            addError.includes('Location'))) &&
+                                        <div className="input">{addError}</div>}
+                                </div>
+                                <div className={addError.includes(
+                                    'identifierType') ? addErrorClass : ''}
+                                >
+                                    <h6><b>Identifier Type</b></h6>
+                                    <select
+                                        className="form-control"
+                                        name="identifierType"
+                                        defaultValue={this.state.addIdentifiers.type}
+                                        onChange={this.handleCreateChange}
+                                    >
+                                        <option value="">Choose type</option>
+                                        {
+                                            this.props.stateData.identifierstypesArray.map(idType => (
+                                                <option key={idType.uuid} value={idType.display}>{idType.display}</option>
+                                            ))
+                                        }
+                                    </select>
+
+                                    {addError.includes('identifierType') &&
+                                        <div className="input">{addError}</div>}
+                                </div>
+                                <div className={addError.includes('Location') ? addErrorClass : ''}>
+                                    <h6><b>Location</b></h6>
+                                    <select
+                                        className="form-control"
+                                        name="location"
+                                        defaultValue={this.state.addIdentifiers.location}
+                                        onChange={this.handleCreateChange}
+                                    >
+                                        <option value="">Choose location</option>
+                                        {
+                                            this.props.stateData.locationArray.map(location => (
+                                                <option key={location.uuid} value={location.display}>{location.display}
+                                                </option>
+                                            ))
+                                        }
+                                    </select>
+                                    {addError.includes('Location') &&
```

Comment:
You may want to fix the indentation here

Reply:
done